doc/rst/technotes/gpu.rst

```diff
@@ -561,6 +561,20 @@ marked with * are covered in our nightly testing configuration.
   * Software:ROCm 4.2*, 4.4, 5.4
 
 
+GPU Support on WSL
+------------------------------------------------
+
+Starting with Chapel 2.0, we have demonstrated the ability to use NVIDIA GPUs
+through WSL. To enable GPU support on WSL we require the CUDA Toolkit to be
+installed in the WSL environment and the NVIDIA driver to be installed on the
+Windows host.
+See the `NVIDIA documentation
+<https://docs.nvidia.com/cuda/wsl-user-guide/index.html#getting-started-with-cuda-on-wsl-2>`_
+for more information on setting up CUDA on WSL.
+See `Using Chapel on WSL <../platforms/windows.html#using-chapel-on-wsl>`_
+for more information on using Chapel with WSL.
+
```

Comment:
... but then maybe add:

> Note that this is not a configuration we test nightly. Please file a bug report if you encounter any issues.

You can link to: https://github.com/chapel-lang/chapel/issues/new